modules/nf-core/wipertools/fastqgather/main.nf

```diff
@@ -0,0 +1,46 @@
+process WIPERTOOLS_FASTQGATHER {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/wipertools:1.1.3--pyhdfd78af_0':
+        'biocontainers/wipertools:1.1.3--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(fastq_in)
+
+    output:
+    tuple val(meta), path("*_gather.fastq.gz") , emit: fastq_out
+    path "versions.yml"                        , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args      = task.ext.args ?: ''
+    def prefix    = task.ext.prefix ?: "${meta.id}"
```

Comment:
I'd suggest this change, where the `_gather` gets added to the default prefix, so it can be overridden if required.
```suggestion
    def prefix    = task.ext.prefix ?: "${meta.id}_gather"
```

Reply:
the problem here is that if the "task.ext.prefix" is passed and does not contain the substring "_gather", with the suggested change below, we get an error with the output file name not matching. Isn't it?

Eventually, I might change:

def prefix    = task.ext.prefix ?: "${meta.id}" 
prefix        = prefix + "_gather"

is that necessary?

Reply:
Sure, so you need to change the pattern to be prefix, as I've just done.
Which requires the prefix variable not to have `def`, but that is fine.

Reply:
At the moment there is no way to make a fastq file that doesn't end in "gather.fastq.gz".

Reply:
changed